samples/openapi3/client/petstore/dart-dio/petstore_client_lib_fake/lib/src/model/cat.dart

```diff
@@ -42,11 +42,6 @@ class _$CatSerializer implements PrimitiveSerializer<Cat> {
     Cat object, {
     FullType specifiedType = FullType.unspecified,
   }) sync* {
-    yield r'className';
```

Comment:
can you please merge the latest master into this branch? there shoudn't be any update to the dart samples as part of this change.